tests/db_functions/json/test_json_array.py

```diff
@@ -0,0 +1,122 @@
+import re
+import unittest
+
+from django.db import NotSupportedError, connection
+from django.db.models import CharField, F, Value
+from django.db.models.functions import Cast, JSONArray, Lower
+from django.test import TestCase
+from django.test.testcases import skipIfDBFeature, skipUnlessDBFeature
+from django.utils import timezone
+
+from ..models import Article, Author
+
+
+@skipUnlessDBFeature("has_json_object_function")
```

Comment:
```suggestion
@skipUnlessDBFeature("supports_json_field")
```

Reply:
is it guaranteed that all backends with `supports_json_field == True` have an implementation for JSONArray? I guess the test suite will tell us